evalml/tests/model_understanding_tests/test_permutation_importance.py

```diff
@@ -335,7 +335,8 @@ def test_fast_permutation_importance_matches_slow_output(
         objective = "R2"
 
     mock_supports_fast_importance.return_value = True
-    parameters["Random Forest Classifier"] = {"n_jobs": 1}
+    parameters["Estimator"] = {"n_jobs": 1}
```

Comment:
This was incorrect before. We need to use the name of the component as it is found in dictionary keys, not the default name of the component.